src/System.Reflection/tests/AssemblyNameTests.cs

```diff
@@ -407,10 +402,8 @@ public void Version(Version version, string versionString)
             assemblyName.Version = version;
 
             string expected = "MyAssemblyName, Version=" + versionString;
-            string extended = expected + ", Culture=neutral, PublicKeyToken=null";
 
-            Assert.True(assemblyName.FullName == expected || assemblyName.FullName == extended,
-                        $"Expected\n{assemblyName.FullName} == {expected}\nor\n{assemblyName.FullName} == {extended}");
+            Assert.Equal(assemblyName.FullName, expected);
```

Comment:
This first argument for `Assert.Equal` is expected, the second argument is actual.